examples/tf/debugging/simple_spread/feedforward/centralised/run_mappo.py

```diff
@@ -88,7 +88,8 @@ def main(_: Any) -> None:
         network_factory=network_factory,
         logger_factory=logger_factory,
         num_executors=1,
-        optimizer=snt.optimizers.Adam(learning_rate=5e-4),
+        policy_optimizer=snt.optimizers.Adam(learning_rate=5e-4),
```

Comment:
Is it standard to use the same learning_rate for the policy and critic?

Reply:
This is not immediately obvious @DriesSmit .

So acme's ddpg uses the same lr - https://github.com/deepmind/acme/blob/58c65c6011a7a05170a41a800c6e80d33d137db8/acme/agents/tf/ddpg/agent.py#L148, mappo on mpe uses the same lr https://github.com/marlbenchmark/on-policy/blob/9677c91ff7de0defc245f65c1ea31c8587df162a/onpolicy/scripts/train_mpe.sh#L14, but this is not always the case.

For now, I think this is okay since it is configurable and this is just for an example.

Reply:
Sounds good to me :+1: